saleor/graphql/product/mutations/products.py

```diff
@@ -525,10 +527,38 @@ def clean_attributes(
         )
         return attributes
 
+    @staticmethod
+    def parse_description_json_to_string(description):
```

Comment:
This is a core utility that could be used in many modules of Saleor, it should be moved out from a mutation. I suggest creating a file `saleor/core/utils/editorjs.py` and moving this function there. We could also move there stuff from `editorjs_sanitizer.py`. Then we will have one file with utilities for editor.js.